tools/src/test/java/org/apache/kafka/tools/GetOffsetShellTest.java

```diff
@@ -182,14 +187,19 @@ public void testTopicPartitionsArg() {
         setUp();
 
         List<Row> offsets = executeAndParse("--topic-partitions", "topic1:0,topic2:1,topic(3|4):2,__.*:3");
-        List<Row> expected = Arrays.asList(
-                new Row("__consumer_offsets", 3, 0L),
+        ArrayList<Row> expected = new ArrayList<>(
+            Arrays.asList(
                 new Row("topic1", 0, 1L),
                 new Row("topic2", 1, 2L),
                 new Row("topic3", 2, 3L),
                 new Row("topic4", 2, 4L)
+            )
         );
 
+        if (!cluster.isKRaftTest()) {
```

Comment:
I feel this test needs to verify the match pattern `__.*:3`, so we can do read for KRaft mode to create the internal topics.

Reply:
Currently we didn't create any internal topic when running in kraft mode, so we can't verify the match pattern `__.*:3`. Shall we create an internal topic when running in Kraft mode ?

Reply:
> Shall we create an internal topic when running in Kraft mode ?

yep, we can create a consumer to read records before verifying the match pattern

Reply:
Sure. I've created Consumer to subscribe and poll from the topic for this test.